dbt/adapters/databricks/impl.py

```diff
@@ -106,6 +108,7 @@
 @dataclass
 class DatabricksConfig(AdapterConfig):
     file_format: str = "delta"
+    table_format: TableFormat = TableFormat.DEFAULT
```

Comment:
In future, we may fold file_format into table_format; adding table_format for consistency with other dbt adapters.